website/docs/core-concepts/stacks/yaml-functions/store.mdx

```diff
@@ -9,20 +9,23 @@ import Intro from '@site/src/components/Intro'
 import Terminal from '@site/src/components/Terminal'
 
 <Intro>
-The `!store` YAML function allows reading the values from a remote [store](/core-concepts/projects/configuration/stores) (e.g. SSM Parameter Store, Artifactory, etc.)
+The `!store` YAML function allows reading the values from a remote [store](/core-concepts/projects/configuration/stores) (e.g. SSM Parameter Store, Artifactory, Redis, etc.)
 into Atmos stack manifests.
 </Intro>
 
 ## Usage
 
-The `!store` function can be called with either two or three parameters:
+The `!store` function can be called with either two or three parameters, and optionally a default value:
 
 ```yaml
   # Get the `key` from the store of a `component` in the current stack
   !store <component> <key>
 
   # Get the `key` from the store of a `component` in a different stack
   !store <component> <stack> <key>
+
+  # Get the `key` from the store of a `component` in a different stack, with a default value
+  !store <component> <stack> <key> | default <default_value>
```

Comment:
Not sure where we should mention that quoted values to default will require also quoting the entire string. YAML doesn't like random quotes placed in the middle of an unquoted string.

Reply:
Like `!env FOO something`

Reply:
This would not work because a string with 4 parts could be:

 `<store_name> <component> <key> <default>`
 `<store_name> <stack> <component> <key>`
 
And we'd have no way to distinguish between them.